data/presets/leisure/bathing_place.json

```diff
@@ -0,0 +1,45 @@
+{
+    "aliases": [
+        "Bathing spot",
+        "Dip spot",
+        "Dipping spot",
+        "Swim spot",
+        "Swimming spot",
+        "Wild swimming spot"
+    ],
+    "icon": "fas-water-ladder",
```

Comment:
"ladder" does not sound ideal for a place that is often unofficial and will likely not have a ladder. What would be other options? 

Do we use ladder for other presets as well? (Might this cause confusion?)

Reply:
Thanks! So
- `public_bath` uses maki-water which is just a drop of water, and the same icon is used by a lot of other water-related features e.g. lake and spring, so probably a poor choice
- `beach_resort` uses [fas-umbrella-beach](https://fontawesome.com/icons/umbrella-beach?f=classic&s=solid) which is probably worse because it doesn't even show water
- `swimming_area` uses fas-swimmer which I can't find on the FAS website but looks identical to [person-swimming](https://fontawesome.com/icons/person-swimming?f=classic&s=solid)
- I suggested [fas-water-ladder](https://fontawesome.com/icons/water-ladder?f=classic&s=solid) because I was under the impression that it was not used in this project (the search didn't return anything) but I have since realised that it's actually being used in the swimming pool preset where it's called fas-swimming-pool!